src/foam/u2/view/TableCellFormatter.js

```diff
@@ -123,6 +123,11 @@ foam.CLASS({
       generateJava: false,
       name: 'tableHeaderFormatter',
       value: function(axiom) { this.add(axiom.label); }
+    },
+    {
+       type: 'Int',
+       name: 'tableWidth',
+       value: 150
```

Comment:
This should probably be a bit lower 125/130. Any reason it is 150?

Reply:
I will try with 125/130 again.

Reply:
I tried with 130. I think you are right. 130 is good. I only need to fix one extra more. Since I already set the table  width in some places indivisually. Thank you!

Reply:
Nice, thanks. Most buttons will only be one or two words so setting default this wide wouldn't make sense